websites/O/Origin/presence.ts

```diff
@@ -0,0 +1,50 @@
+const presence = new Presence({
+    clientId: "760611329588985886"
+  }),
+  presenceData: PresenceData = {
+    largeImageKey: "icon"
+  };
+
+presence.on("UpdateData", async () => {
+  if (document.location.pathname == "/my-home") {
+    presenceData.details = "Browsing the homepage";
+  } else if (document.location.pathname.startsWith("/store/download")) {
+    presenceData.details = "Installing the Origin launcher";
+    if (document.location.pathname.startsWith("/store")) {
+    presenceData.details = "Browsing for a game";
+  } else if (document.location.pathname.startsWith("/profile/achievements")) {
+    presenceData.details = "Viewing their achievement progress";
+  } else if (document.location.pathname.startsWith("/eaAccounts")) {
+    presenceData.details = "Editing their settings";
+  } else if (document.location.pathname.startsWith("/game-library")) {
+    presenceData.details = "Browsing their game library";
+  } else if (document.location.pathname.startsWith("/search")) {
+    presenceData.details = "Searching the store";
+  } else if (document.location.pathname.startsWith("/profile")) {
+    presenceData.details = "Viewing a profile";
+  } else if (document.location.pathname.startsWith("/profile")) {
+    presenceData.details = "Looking through their friends list.";
+  } else if (document.location.pathname.startsWith("profile/wishlist")) {
+    presenceData.details = "Viewing their wishlist";
+  } else if (document.location.pathname.startsWith("profile/friends")) {
+    presenceData.details = "Viewing their friends list";
+  };
+  var profileData: PresenceData = {
+      details: "Viewing " + name + "'s Profile",
+      largeImageKey: "logo"
+    };
+    
+  var storeData: PresenceData = {

```

Comment:
no var, and then you can merge these consts

Reply:
? I'm confused.

Reply:
You shouldnt use var, you should use let or const, in this case const, and in the case you can also merge the consts

const var1 = obj,
var2 = obj2;

Reply:
oh ok 👍🏻

Reply:
Yeah, iv'e hit a wall.

Reply:
it's const var {
details blah blah
imagekey
} right?

Reply:
or am I not getting this at all xD